### DISCUSSION TITLE
Was 23.3.0 re-released with new binaries?

### DISCUSSION THREAD

**Discussion by GUI** [dbeaver/dbeaver#22172] [DBeaver Development]
I'm puzzling over why some of the DBeaver binary checksums seem to have changed for the 23.3.0 release yesterday, and just wanted to make sure this was intentional and nothing we should be worried about.

It looks like 23.3.0 was [released on 2023-12-03](https://dbeaver.io/2023/12/03/dbeaver-23-3-0/) and that jives with the latest activity on the [git tag](https://github.com/dbeaver/dbeaver/tree/23.3.0). However, all of the [current 23.3.0 files](https://dbeaver.io/files/23.3.0/) appear to have been replaced on 2023-12-12 and the same with the [GitHub Release](https://github.com/dbeaver/dbeaver/releases/tag/23.3.0).

Since I can't find any mention of changes, I just wanted to make sure this was intentional, since it's caused checksum mismatches for the previous 23.3.0 release files (https://github.com/Homebrew/homebrew-cask/pull/162379), and I wanted to ensure that the latest binaries should be trusted. Thanks!

**Comment by LonwoLonwo** [comment]
Hello @GUI 

Yes, we had a rerelease due to the critical bug found in the program.
This was intentional and nothing you should be worried about, yes.

Thanks for the attention shown.
